CHANGELOG.md

```diff
@@ -4,8 +4,16 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ## Unreleased
 
+### Fixes
```

Comment:
I think I'm not understanding something. What release is this? Not 5.3.1?

Reply:
Nah, this one just missed out and was merged after the release.

Reply:
Ah! A rogue